BlueprintUI/Sources/Layout/LayoutAttributes.swift

```diff
@@ -88,6 +93,9 @@ public struct LayoutAttributes {
         view.alpha = alpha
         view.isUserInteractionEnabled = isUserInteractionEnabled
         view.isHidden = isHidden
+        if let tintAdjustmentMode {
+            view.tintAdjustmentMode = tintAdjustmentMode
+        }
```

Comment:
I think we still need to update this if `tintAdjustmentMode` becomes `nil`, but it's unclear what to utilize as the default value, or if that's even appropriate. The docs seem to imply that this will traverse the subview hierarchy to determine a value unless you set this value, but there doesn't appear to be a way to revert it back to the subview traversal behavior. Am I understanding that correctly?

Reply:
Or is `automatic` the same as the "default" state?

Reply:
I'm seeing views in the Market Catalog app default to `UIViewTintAdjustmentMode(rawValue: 1)`, where `case normal = 1`, which I wasn't expecting... I don't see us overriding it anywhere.

Reply:
I've never used this property before, but to me the docs seem to imply that you can set this property to `automatic` to get default behavior, but querying this property will never return a value of `automatic`. So, I think I agree that we'll either need to treat `nil` as `automatic`, or make this attribute non-optional with a default of `automatic`.

Reply:
That makes sense...

I think I'd lean towards non-optional with a default of `automatic` in that case.

Reply:
Found this in the UIKit docs.
> -tintAdjustmentMode always returns either UIViewTintAdjustmentModeNormal or UIViewTintAdjustmentModeDimmed. The value returned is the first non-default value in the receiver's superview chain (starting with itself).
> If no non-default value is found, UIViewTintAdjustmentModeNormal is returned.
> When tintAdjustmentMode has a value of UIViewTintAdjustmentModeDimmed for a view, the color it returns from tintColor will be modified to give a dimmed appearance.
> When the tintAdjustmentMode of a view changes (either the view's value changing or by one of its superview's values changing), -tintColorDidChange will be called to allow the view to refresh its rendering.

And for `.automatic`: https://developer.apple.com/documentation/uikit/uiview/tintadjustmentmode/automatic
> The tint adjustment mode of the view is the same as its superview’s tint adjustment mode (or UIViewTintAdjustmentModeNormal if the view has no superview).

So I think this means the default value we provide should be `.automatic`. But it's still not clear to me then how the `LayoutAttributes` concatenation should work so we don't mess up the superview's chain of `tintAdjustmentMode` values. IDK. Perhaps we revert this commit and just tie this property specifically to `Image` like the original commit had so the superview chain isn't polluted.

Reply:
`tintAdjustmentMode` can never return `.automatic`